### DISCUSSION TITLE
Welcome to Faust Discussions!

### DISCUSSION THREAD

**Discussion by sletz** [grame-cncm/faust#583] [General]
## 👋 Welcome!
  **Faust Discussions** is a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Sonder1966** [comment]
Hello, I am new here just wondering can faust export as VST?

**Comment by sletz** [comment]
On with os? You can possibly use the vst/vsti export on Faust Web IDE (macOs, Linux, Windows)
![image](https://user-images.githubusercontent.com/1430487/141963240-5963238a-bd2d-433c-a4d8-c3f8b31545c0.png) or just the JUCE export then compile a VST from the JU<cE project.

**Comment by Sonder1966** [comment]
Many thanks, that worked, it shows up as (Grame), is that correct?

**Reply by sletz** [reply]
Yes.

**Comment by Sonder1966** [comment]
Many thanks, its really great I just discovered it, thank you very much.
